catalog/files/impl/src/main/java/org/projectnessie/catalog/files/adls/AdlsObjectIO.java

```diff
@@ -50,7 +50,7 @@ public void ping(StorageUri uri) throws IOException {
 
     DataLakeFileSystemClient fileSystem = clientSupplier.fileSystemClient(location);
     try {
-      fileSystem.getAccessPolicy();
+      fileSystem.getProperties();
```

Comment:
Why this change? This `ping` function is intended to hit ADLS.

Reply:
I will have permission error with `getAccessPolicy` as my company doesn't allow getting access policy.

Reply:
Fair enough.

Reply:
Maybe move this change to a separate PR for clarity as it relates not to credentials but to server-side policies?

Reply:
#9134